baselines/train/run_ray_train.py

```diff
@@ -53,10 +53,11 @@ def get_cli_args():
   parser.add_argument(
       "--exp",
       type=str,
-      choices = ['pd_arena','al_harvest','clean_up','territory_rooms','day_care','commons_harvest__partnership','commons_harvest__open','commons_harvest__private_property_pc','commons_harvest__private_property','commons_harvest__open_disable_zapping','commons_harvest__closed'],
+      choices = ['pd_arena','al_harvest','clean_up','territory_rooms','day_care','commons_harvest__partnership','commons_harvest__open','commons_harvest__private_property_pc','commons_harvest__private_property','commons_harvest__open_disable_zapping','commons_harvest__closed', 'commons_harvest__open_abundance'],
       default="pd_arena",
       help="Name of the substrate to run",
-  )
+  ) #Todo, fetch this list of choices from elswhere? stuff like this is a maintence headache.
```

Comment:
Agreed that all occurrences of these experiment/substrate names can be centralized, as opposed to manually updating these everywhere

Should we track this on Slack instead of just leaving the comment here?

Reply:
Not sure if its easy to store this sort of info in slack, given that we have just one channel.  Also not sure if this refactoring is worth doing (in so far as the timeline we have so aisf).